lib/bunq/user.rb

```diff
@@ -63,6 +63,10 @@ def notification_filter_url
       Bunq::NotificationFilterUrl.new(@resource)
     end
 
+    def credential_password_ip
```

Comment:
Usually this is named in plural form, so `credential_password_ips`, unless it represents a single thing (in which case typically an `id` is provided as an argument). 

For example see `card(id)` and `cards`.